lib/qops/cookbook/cookbook.rb

```diff
@@ -130,10 +130,19 @@ def config
   end
 
   def s3
-    @s3 ||= Aws::S3::Client.new(
-      region: 'us-east-1',
-      credentials: config.opsworks.config.credentials.credentials
-    )
+    @s3 ||= Aws::S3::Client.new(**aws_configs)
+  end
+
+  def aws_configs
+    aws_config = { region: 'us-east-1' }
+    # use the profile if found
+    if options[:profile]
```

Comment:
you can use one-liner like this to make this simple: aws_config = options[:profile] ? A : B